test/user.js

```diff
@@ -98,6 +98,101 @@ describe('User', () => {
     });
 
   });
+  /*
+  * Testing the /users/sessions (POST) route
+  */
+  describe('/POST users/sessions', () => {
+
+    let newUser = {
+      name: 'Kevin',
+      lastName: 'Temes',
+      email: 'kevin.temes@wolox.com.ar',
+      password: '12345678'
+    };
+
+    it('should successfully log a user', (done) => {
+
+      let correctLogin = {
+        email: 'kevin.temes@wolox.com.ar',
+        password: '12345678'
+      };
+
+      chai.request(server)
+        .post('/users')
+        .send(newUser)
+        .end((err, res) => {
+
+          chai.request(server)
+            .post('/users/sessions')
+            .send(correctLogin)
+            .end((err, res) => {
+              res.should.have.status(200);
+              dictum.chai(res, 'User signin');
+              done();
+            });
+
+        });
+
+    });
+
+    it('should throw an error when attempting a login with empty or null email/password', (done) => {
+
+      let emptyLogin = {
```

Comment:
Remember if you are not re-assigning a var you should use const instead of let, we should configure the linter to warn you about this.